docs/docs/term_index.xml

```diff
@@ -147,6 +147,9 @@
          <term file="ml.html" name="loss_mean_squared_multioutput_" include="dlib/dnn.h"/>
          <term file="ml.html" name="loss_mmod_" include="dlib/dnn.h"/>
          <term file="dlib/dnn/loss_abstract.h.html" name="mmod_options" include="dlib/dnn.h"/>
```

Comment:
This is what I was referring to.

Reply:
Oh yeah, I forgot some of these :|

Update your git though, I did add the loss_yolo to ml.xml.  You must be looking at a branch or something.

Reply:
Oh, I don't know how that happened! Fixed :)